i18n.json

```diff
@@ -0,0 +1,7 @@
+{
+  "locales": ["en", "ja"],
+  "defaultLocale": "en",
```

Comment:
Why not default to Japanese? Most users will be Japanese speakers.

(Yes, I know I'm using English now 😊)

Reply:
Indeed.
I will change this.
(Then, why did "omise" default to English...?🤔)

Reply:
Thank you. I think those were different times for Henkaku, too.

Also as engineers, we want English activity on our GitHub profile, so it's easy to just default to English for everything. And it's harder to change something that has started already.

Reply:
I see.
I too think the better the user experience, so I will change it.

Reply:
Thank you!